scripts/jest/config.build.js

```diff
@@ -31,12 +31,20 @@ moduleNameMapper[
 
 // Map packages to bundles
 packages.forEach(name => {
-  // Root entry point
-  moduleNameMapper[`^${name}$`] = `<rootDir>/build/node_modules/${name}`;
-  // Named entry points
-  moduleNameMapper[
-    `^${name}/(.*)$`
-  ] = `<rootDir>/build/node_modules/${name}/$1`;
+  if (name === 'scheduler') {
+    // Scheduler is a special case because we mock it by default, but unmock
+    // for specific modules.
+    moduleNameMapper['^scheduler$'] = `<rootDir>/build/node_modules/scheduler`;
+    moduleNameMapper['^scheduler/tracing$'] =
+      '<rootDir>/build/node_modules/scheduler/tracing';
```

Comment:
How is this not just the same as below? Can you get rid of this special case?

Reply:
Yeah, got it working by changing the regex to only match the top level directory per package.